charts/core/README.md

```diff
@@ -58,66 +84,92 @@ Chart container component.
+
 ### `Line`
 
 Component that takes in data points and renders a single line on the chart.
 
 #### Props
 
-| Name   | Description                                                                                                              | Type                                                        | Default |
-| ------ | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------- | ------- |
-| `name` | Unique name used to identify the series. **Important note:** If two lines have the same name, only one will be rendered. | string                                                      |         |
-| `data` | Data array of tuples that represent x and y coordinates in the series.                                                   | Array<[string \| number \| Date, string \| number \| Date]> |         |
+| Name   | Description                                                                                                              | Type                                                          | Default |
+| ------ | ------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------- | ------- |
+| `name` | Unique name used to identify the series. **Important note:** If two lines have the same name, only one will be rendered. | `string`                                                      |         |
+| `data` | Data array of tuples that represent x and y coordinates in the series.                                                   | `Array<[string \| number \| Date, string \| number \| Date]>` |         |
+
+### `Header`
+
+Component for rendering a header in a chart.
+
+#### Props
+
+| Name                         | Description                                                                                                                                           | Type              | Default |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- | ------- |
+| `title`                      | The title to display in the chart header.                                                                                                             | `string`          |         |
+| `showDivider` _(optional)_   | When true, renders a dividing line on top of header. Useful when stacking charts, such as in a `ChartGroup`.                                          | `boolean`         |         |
+| `headerContent` _(optional)_ | Content that will be rendered to the right of the title. Useful for adding components such as `IconButton`'s that control functionality in the chart. | `React.ReactNode` |         |
```

Comment:
nit: I didn't see a story that showcases this feature. Would you consider adding one to ensure that it works?

Reply:
Absolutely. I'm curious how we demo something like this since it takes in a `ReactNode`. Any suggestions?

Reply:
Code might be a good example: https://storybook.mongodb.design/?path=/story/components-code--with-custom-actions.

I think you would need to have individual stories for standalone components.

Reply:
I ended up doing a pretty big refactor here. I separated things into 2 stories "Chart" and "ChartCard", with the following stories:
- Chart
  - Basic
  - With Header Content 

- ChartCard
  - Basic
  - With Header Content
  - Containing Chart
  - Containing Multi Charts

I'd rather not do it in this PR (because folks are waiting to use this), but I actually think this is how the packages should ultimately be split as well. `@lg-charts/core` should probably really be `@lg-charts/chart` and only export what's needed for a chart itself. `@lg-charts/chart-card` should be separate and only export the card.

Reply:
sounds good to me